doc/stdlib/math.rst

```diff
@@ -813,7 +813,12 @@ Mathematical Functions
 
 .. function:: factorial(n)
 
-   Factorial of n
+   Factorial of ``n``.  If ``n`` is an integer type, the factorial is
+   computed as an integer (promoted to at least 32 bits).  Note that
+   this may overflow if ``n`` is not small, but you can use
+   ``factorial(big(n))`` to compute the result exactly in arbitrary
+   precision.  If ``n`` is not an integer, this is equivalent to
```

Comment:
is not an **``` func:`Integer` ```**?

(because we are talking about the Julia abstract type, not the math concept where `1.0` is considered an integer)